RFC-003-SWAP-Basic.md

```diff
@@ -88,8 +88,8 @@ How to construct an HTLC for each ledger will be defined in subsequent RFCs.
 ## Setup Phase
 
 In the setup phase the two parties exchange [RFC002](RFC-002-SWAP.adoc) SWAP messages to negotiate the parameters of the HTLCs.
-The values α, β, **A** and **B** used below refer to the ledgers and assets described by the SWAP headers `alpha_ledger`, `beta_ledger`, `alpha_asset` and `beta_asset` respectively.
-Additionally, α-HTLC and β-HTLC refer to the HTLCs deployed on the α and β ledgers.
+The values alpha, beta, **A** and **B** used below refer to the ledgers and assets described by the SWAP headers `alpha_ledger`, `beta_ledger`, `alpha_asset` and `beta_asset` respectively.
```

Comment:
:upside_down_face: I'd say we can probably get rid of **A** and **B** too and generally refer to things by their full name for clarity: alpha ledger, beta ledger, alpha asset, beta asset. Could be done in a separate PR anyway.

Reply:
I guess 3 people liking this idea means I gotta do it in this PR :D 

/cc @bonomat @D4nte